mechanics/CREATING-A-SANDBOX-REPO.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Creating a sandbox repo"
+linkTitle: "Sandbox repo process"
+weight: 30
+type: "docs"
+---
+
+# Background
+
+The [`knative-sandbox`](https://github.com/knative-sandbox) GitHub org exists to
+hold "non-core" Knative components which are owned and sponsored by a Working
+Group but are not part of an official Knative release.
+
+## Criteria
+
+A Working Group Lead (either
+[technical](../ROLES.md#working-group-technical-lead) or
+[execution](../ROLES.md#working-group-execution-lead)) may request a new repo in
+`knative-sandbox` from any [TOC member](../TECH-OVERSIGHT-COMMITTEE.md).
+Generally, the request will be granted, though the TOC may have additional
+questions or suggest an alternate mechanism in some cases.
+
+Some of the following steps may require permissions that are only available to
+the TOC or Steering Committee, though others are largely self-service or require
+other WGs to review and approve impacting changes.
+
+### Technical requirements
+
+- Any contributed code should be contributed by the original authors or
+  copyright holders under an Apache 2.0 license.
+
+- Names between `knative-sandbox` and the main `knative` repo should not
+  overlap. This simplifies promoting repos between the two orgs and setting up
+  `knative.dev` import paths for golang.
+
+- Prow automation for tests is not necessarily required for `knative-sandbox`,
+  but the Google CLA bot and OWNERS files/tide merge should be enforced.
+
+## Process
+
+1. (Requires Org owner) Create the new repo in
+   https://github.com/knative-sandbox using the "New" button. Set the repo to
+   public and include an "Apache License 2.0" but no `.gitignore` or `README`.
```

Comment:
Depending on the purpose of the repo, we will sometimes "use template" on the sample-controller or sample-source repo.

Once the repo is created, the repo must be added to the list of repos administered by "Knative Admin" (which grants the Prow bots access).

Reply:
for the ACL, I now see the comment below.